tests/models/convnext/test_modeling_convnext.py

```diff
@@ -137,6 +137,7 @@ class ConvNextModelTest(ModelTesterMixin, unittest.TestCase):
         else ()
     )
 
+    fx_ready = True
```

Comment:
This flag does not exist in the lib. Did you mean `fx_compatible`?

Reply:
Yes, my bad!